test/link-style-autolink-only.md

```diff
@@ -0,0 +1,20 @@
+# Autolink Link Style
```

Comment:
I try to have the title match the file name. I think you've got separate files for each setting (great!), but this would read "Link Style Autolink Only" in my convention. A couple of the others look to be copy/pasted without updates.